source/docs/articles/local/cli/machine-tokens.md

```diff
@@ -21,10 +21,17 @@ Machine tokens:
 
 ![Machine token ready modal](/source/docs/assets/images/machine-token-ready.png)
 ## Authenticate into Terminus
+
 Use your token to authenticate into Terminus by running the following command:  
-```bash
-terminus auth login --machine-token=<machine token>
-```
+`terminus auth login --machine-token=<machine token>`
+
+## Switch Between Multiple Pantheon User Accounts
+
+Machine tokens are paired with your email address (Pantheon user account), so it is possible to use multiple user accounts/tokens on one machine.  
```

Comment:
Suggest replacing this paragraph with: 
> Since tokens are keyed to the email address associated with your Pantheon user account, you can easily switch between user accounts: